README.md

```diff
@@ -92,45 +99,129 @@ like so:
     % cd halide_build
     % make -f ../Halide/Makefile
 
-#### Building Halide with cmake
+Building Halide with CMake
+==========================
+
+### MacOS and Linux
 
-If you wish to use cmake to build Halide, the build procedure is:
+Follow the above instructions to build LLVM or acquire a suitable binary release.
+Then create a separate build folder for Halide and run CMake, pointing it to your LLVM installation.
 
-    % mkdir cmake_build
-    % cd cmake_build
-    % cmake -DLLVM_DIR=/path-to-llvm-build/lib/cmake/llvm -DCMAKE_BUILD_TYPE=Release /path/to/halide
-    % make -j8
+    % mkdir Halide-build
+    % cd Halide-build
```

Comment:
Nit: Should we make Ninja the default recommendation here, rather than Make? (IMHO it is superior, but I wonder if it would be less-surprising to out users to assume they are using Make, and maybe add a comment that "of course it works with Ninja too"). Or is Ninja basically required for Windows usage?

Reply:
> Should we make Ninja the default recommendation here, rather than Make?

Debatable. I like it much better for the faster incremental builds and cleaner output. It's also the only build system that works just as well on all major platforms.

> Or is Ninja basically required for Windows usage?

It is not required on Windows at all. I've tested with both the Ninja and the Visual Studio generators.

Reply:
Maybe just add text to the effect of what you said here ("We recommend Ninja because blah, but Make and MSBuild work fine too")

Reply:
I dropped the `-G Ninja` in the main command and just mentioned that it's an option for Mac OS & Linux.